contract_manager/src/evm.ts

```diff
@@ -0,0 +1,92 @@
+import Web3 from "web3";
```

Comment:
hmm we should standardize on web3.js or ethers -- suggest asking the team what they prefer. I usually pick ethers (i had some issues with web3.js like 2 years ago and have stuck with ethers since), but i'm not the most informed on this question.